bears/css/PostCSSBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
+
+
+class PostCSSBear(Lint, LocalBear):
+    executable = "postcss"
+    diff_message = "PostCSS plugins in action."
+    gives_corrected = True
+
+    def run(self, filename, file, postcss_plugins: typed_list(str)=[]):
+        '''
+        This bear adds ``PostCSS`` utilities to analyze CSS code.
+
+        :param postcss_plugins:     List of PostCSS plugins to use with
+                                    this bear which can be defined in
+                                    .coafile.
+        '''
+        if postcss_plugins:
+            for plugin in postcss_plugins:
```

Comment:
Ahh, nice. Let's use that then ?
Also, what happens if I give `--use plugin1 --use non-existent-plugin` ?
if there's an error message can we show that ?

On Wed, Mar 23, 2016 at 2:31 PM, Karan Sharma <notifications@github.com>
wrote:

> In bears/css/PostCSSBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/283#discussion_r57126549>
> :
>
> > +
> > +class PostCSSBear(Lint, LocalBear):
> > +    executable = "postcss"
> > +    diff_message = "PostCSS plugins in action."
> > +    gives_corrected = True
> > +
> > +    def run(self, filename, file, postcss_plugins: typed_list(str)=[]):
> > +        '''
> > +        This bear adds ``PostCSS`` utilities to analyze CSS code.
> > +
> > +        :param postcss_plugins:     List of PostCSS plugins to use with
> > +                                    this bear which can be defined in
> > +                                    .coafile.
> > +        '''
> > +        if postcss_plugins:
> > +            for plugin in postcss_plugins:
>
> The format is --use plugin_1 --use plugin_2
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/283/files/206c67d1e6e14fa66794e42fc942c644c8a00f50#r57126549>
>